packages/web-api/src/file-upload.ts

```diff
@@ -190,7 +191,7 @@ export async function getFileData(options: FilesUploadV2Arguments | FileUploadV2
 
 export function getFileDataLength(data: Buffer): number {
   if (data) {
-    return Buffer.byteLength(data, 'utf8');
+    return Buffer.byteLength(data);
```

Comment:
🔗 https://nodejs.org/api/buffer.html#static-method-bufferbytelengthstring-encoding

Changed this to prefer the provided `Buffer` type with a default "utf8" for **strings** which is found in all LTS versions.

Reply:
📣 Reverted in https://github.com/slackapi/node-slack-sdk/pull/2236/commits/f141ff0a937d75a32fbfb5f36e3e02cf00d13be0 since the included default was not breaking tests AFAICT!